package.json

```diff
@@ -635,22 +646,22 @@
                 },
                 {
                     "command": "makefile.outline.buildTarget",
-                    "when": "view == makefile.outline",
+                    "when": "makefile:fullFeatureSet && view == makefile.outline",
```

Comment:
The following adds of "makefile:fullFeatureSet" are needed now because the UI is visible now even if fullFeatureSet is false, while before it was not visible, so no extra reason to hide all its components if the whole panel was hidden.

Reply:
To make sure I understand this right: 
The goal is to make the UI visible in all cases. However, there are a certain subset of features that will not be visible or available unless `fullFeatureSet` is set to `true`?

Reply:
A follow up question: What is the purpose of the UI being visible now even if `fullFeatureSet` is false? Would there be buttons or things that are now disabled due to `fullFeatureSet` being false?

Reply:
The UI needs to be visible now in order to see the status (make or makefile not found). Otherwise this approach (a status in this UI) would be useless because the same cause for the status to be reported used to cause the UI to hide.
So now the UI is visible but all buttons that before didn't need to hide, are not visible now.
Another reason to make sense for the UI to be accessible is that the UI is one entry point for correcting the problem that caused the undesired situation: there are edit buttons (pen icons) to open the settings and change "makefile.makePath" and "makefile.makefilePath" settings.